PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -835,8 +835,8 @@ def getMETUncertainties(self, process, metType, metModName, electronCollection,
         if not hasattr(process, "pfCandsForUnclusteredUnc"+postfix):
 
             #Jet projection ==
-            pfCandsNoJets = cms.EDProducer("CandPtrProjector", 
-                                           src = pfCandCollection, 
+            pfCandsNoJets = cms.EDProducer("CandPtrProjector",
+                                           src = cms.InputTag("puppiForMET") if self.getvalue("Puppi") else copy.copy(self.getvalue("pfCandCollection")),
```

Comment:
this line is actually a change of sintax
from pfCandCollection to copy.copy(self.getvalue("pfCandCollection"))
[ might be the source of unexpected changes of this PR ]

Reply:
@alefisico 

this line is responsible of the changes in the previous year i.e. in the EE17

please change from 

`src = cms.InputTag("puppiForMET") if self.getvalue("Puppi") else copy.copy(self.getvalue("pfCandCollection")),`

to 

`src = cms.InputTag("puppiForMET") if self.getvalue("Puppi") else pfCandCollection`